lib/connectors/sentinel_connector.js

```diff
@@ -47,27 +47,26 @@ SentinelConnector.prototype.connect = function (callback, eventEmitter) {
     _this.currentPoint += 1;
     if (_this.currentPoint === _this.sentinels.length) {
       _this.currentPoint = -1;
+      if (lastError) {
```

Comment:
If there's no `lastError`, shouldn't we just connect to the first sentinel again instead of just returning?

Reply:
We may set `lastError` to `null` when resolved successfully.

Reply:
Thanks for the check and your nice help.